assets/css/map/controls/_street_view_switch.scss

```diff
@@ -4,6 +4,7 @@
   // Leaflet uses px for positioning of their controls so we use px here to match
   top: 10px;
   right: 55px;
+  z-index: 1000; // Same control layer as leaflet controls
```

Comment:
[Same as previous comment.]

Reply:
I see there is [$max-leaflet-z-index](https://github.com/mbta/skate/blob/e0116fb6190c6a34b12c2b7343fd8805a4e1f56b/assets/css/_leaflet.scss#L2), maybe that could be used here or expanded into a sass-map

Reply:
PTALA 89d2fca7477460c21728350a6e6a59d651873f0b

Reply:
Further conversation happening here https://github.com/mbta/skate/pull/2117#discussion_r1254508316, resolving this comment